tests/dbt/test_dbt_commands.py

```diff
@@ -56,8 +57,21 @@ class TestDBTDeploy:
     @pytest.fixture
     def dbt_project_path(self, tmp_path_factory):
         source_path = tmp_path_factory.mktemp("dbt_project")
-        dbt_file = source_path / "dbt_project.yml"
-        dbt_file.touch()
+        (source_path / "dbt_project.yml").touch()
+        with open((source_path / "dbt_project.yml"), "w") as f:
+            yaml.dump({"profile": "dev"}, f)
```

Comment:
```suggestion
        dbt_project_cnf = source_path / "dbt_project.yml"
        dbt_project_cnf.write_text(yaml.dump({"profile": "dev"}))
```

This should do the trick